views/admin/users.erb

```diff
@@ -91,11 +90,6 @@
         <tbody>
         <% users.each do |user| %>
             <tr>
-                <td style="max-width: 80px;">
```

Comment:
Since we're removing the image column from the users table here and the sponsors said they also want to remove the tool authorization column would you be able to remove that one here too?